Misc/STAR_DIKRO Box.txt

```diff
@@ -0,0 +1,6 @@
+;DIKRO Box
+DIKRO Box	N050.00.00.000 W001.58.42.919 N049.35.57.962 W001.51.19.004
+DIKRO Box	N049.35.57.962 W001.51.19.004 N049.34.52.873 W000.54.48.094
+DIKRO Box	N049.34.52.873 W000.54.48.094 N049.55.42.580 W000.14.57.328
+DIKRO Box	N049.55.42.580 W000.14.57.328 N050.00.00.012 W000.14.59.838
+DIKRO Box	N050.00.00.012 W000.14.59.838 N050.00.00.000 W001.58.42.919
```

Comment:
```suggestion
DIKRO Box                 N050.00.00.000 W001.58.43.000 N049.35.57.960 W001.51.19.000
DIKRO Box                 N049.35.57.960 W001.51.19.000 N049.34.52.870 W000.54.48.100
DIKRO Box                 N049.34.52.870 W000.54.48.100 N049.55.41.000 W000.15.00.000
DIKRO Box                 N049.55.41.000 W000.15.00.000 N050.00.00.000 W000.15.00.000
DIKRO Box                 N050.00.00.000 W000.15.00.000 N050.00.00.000 W001.58.43.000
```

Reply:
I also changed some of the coordinates @Bucklerz to make them line up with other lines a bit better, and to tidy them up. Could you review/commit please?

Reply:
Looks good, done that now.